scripts/local/up.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
please don't commit this file, the docker environment should be used to make sure testing is consistent across all of our environments.

Reply:
I guess docker should be used in CI but not in local development. does it hurt to have it there?

Reply:
The docker environment is configured so that you can run your tests locally against the containerized test environment. If the supported scripts aren't working we should fix them instead of creating additional scripts which aren't supported.